### ISSUE TITLE
Imagenet pretrain loss and Acc in mobilenetv2

### ISSUE THREAD

**Issue by SteveTsg** [pytorch/examples#691]
After 90 epochs，I got the loss in test is 1.59. The Acc1 is 62, also in test. My question is, the result is right? I used the mobilenetv2. what I want to do is pretrain a new mobilenetv2 with new architecture.
Should I run next 90 epochs?
